src/components/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC, HTMLAttributes, useState, ReactNode } from 'react'
+import { ElToolTipChild, ElToolTipContainer, ElToolTipLabel } from './styles'
+import { useId } from '#src/storybook/random-id'
+
+export interface ToolTipProps extends HTMLAttributes<HTMLDivElement> {
+  children: ReactNode
+  label?: string
+  tip: string
```

Comment:
**suggestion:** Figma uses `description` as the prop name. I think that's probably more appropriate as I'd think "tip" includes the label and the description.